mgmt/src/grpc/converter/old_impl.rs

```diff
@@ -306,8 +306,16 @@ pub fn convert_interface_to_interface_config(
         gateway_config::config::IfType::Vtep => {
             let local = match iface.ipaddrs.as_slice() {
                 [] => Err("VTEP interface requires an IP address".to_string()),
-                [addr] => IpAddr::from_str(addr)
-                    .map_err(|_| format!("Invalid local IP address for VTEP: {addr}")),
+                [addr] => {
+                    let (ip, mask) = parse_cidr(addr)
+                        .map_err(|_| format!("Invalid local IP address for VTEP: {addr}"))?;
+                    if mask == 32 {
+                        Ok(IpAddr::from_str(&ip)
+                            .map_err(|_| format!("Invalid local IP address for VTEP: {addr}")))
+                    } else {
+                        Err("VTEP interface requires a /32 IP address".to_string())
+                    }?
```

Comment:
Should we use a checked type for the VTEP IP?

Reply:
We should be using Prefix to parse this, but I didn't address this here, I just replicated the pattern that was already there.

I have one more cleanup PR that I want to do where I can address this.

Reply:
Nevermind, I pushed a fix to use Prefix to parse the addr in this case.

Reply:
Oh, and as for a checked type for /32, I don't think that is generally needed.  The check is here because we are abusing the interface IP addresses for the VTEP interface.

We should fix this, but it isn't worth the time really because of the changes to the gRPC and how that percolates through the code.

Reply:
OK